flow/__init__.py

```diff
@@ -26,17 +26,12 @@
 from .util.misc import redirect_log
 from .operations import with_job
 
-from signac.common import six
 # Import packaged environments unless disabled in config:
 from .util.config import get_config_value
 if get_config_value('import_packaged_environments', default=True):
     from . import environments  # noqa:F401
 
 
-if six.PY2:
-    raise RuntimeError("Python 2 is not supported by this version of signac-flow!")
```

Comment:
I know that I initially advocated against this, but the project is basically non-functional on Python 2 with all of these changes. There should at least be a very highly visible warning.

Reply:
Personally I don't think this is important. You can't install the package with Python 2 anymore, so why worry about that? You can get another opinion from @vyasr or keep it if you're so inclined. It's not clear to me when (if ever) that warning could go away. We also don't support Python 3.4...

Reply:
Fair enough.